components/director/internal/domain/tenant/repository.go

```diff
@@ -87,13 +95,20 @@ func (r *pgRepository) ExistsByExternalTenant(ctx context.Context, externalTenan
 func (r *pgRepository) List(ctx context.Context) ([]*model.BusinessTenantMapping, error) {
 	var entityCollection EntityCollection
 
-	conditions := repo.Conditions{
-		repo.NewEqualCondition(statusColumn, Active),
+	persist, err := persistence.FromCtx(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "while fetching persistence from context")
 	}
 
-	err := r.listerGlobal.ListGlobal(ctx, &entityCollection, conditions...)
+	prefixedFields := strings.Join(str.PrefixStrings(tableColumns, "t."), ", ")
+	query := fmt.Sprintf(`SELECT DISTINCT %s, ld.tenant_id IS NOT NULL AS %s
```

Comment:
Yes, it was agreed to work like this. However, you made a valid point - maybe it would be better to call it "initialized", not "in use". What do you think?

Reply:
IMO `initialized` would work better, since I've associated `inUse` immediately with our `tenants in use` metrics, and it's not the same, so I'd go with that!